app/services/reports/children_served_report_service.rb

```diff
@@ -40,11 +35,56 @@ def average_children_monthly
     # @return [Float]
     def per_child_monthly
       organization
-        .distributions
-        .for_year(year)
-        .joins(line_items: :item)
-        .merge(Item.disposable)
-        .average('COALESCE(items.distribution_quantity, 50)') || 0.0
+      .distributions
+      .for_year(year)
+      .joins(line_items: :item)
+      .merge(Item.disposable)
+      .average('COALESCE(items.distribution_quantity, 50)') || 0.0
+    end
+
```

Comment:
I'm also not convinced the original is right in the first place.   If you, say, distributed 50 / newborn but 25 for size 6's , the real average would be quite different depending on how many of each you distributed.    I think I'll cycle back with the people who the report is for on that -- I don't think we can easily get an *accurate* number, so I'm going to suggest that either we remove it, or at least put "estimated".

Reply:
For clarity -- I'm going to suggest that to the stakeholder who consumes the report.

Reply:
Nothing to do here until I hear back from them.

Reply:
@jadekstewart3 I've talked to the people who consume the report, and we've come to the conclusion that we should eliminate the "per child monthly"  item on the report.     We can't get a good, non-confusing number for it, and when talking to the stakeholders, we came to the conclusion that it's a number that the banks "know"  better than the system will.  Phew! 

So, it may be beginning to feel like scope drift, but could you remove that line from the report, and remove the 'per child monthly" method -- if it's used for something else,  touch base and I'll dig further.

Reply:
@cielf Whoohoo! Removed and pushed!